postgres-appliance/launch.sh

```diff
@@ -26,13 +33,19 @@ for i in $(seq 0 7); do
         touch "${PGLOG}/postgresql-$i.csv"
     fi
 done
-chown -R postgres:postgres "$PGROOT" "$RW_DIR/postgresql"
+chown -R postgres: "$PGROOT" "$RW_DIR/certs"
 chmod 01777 "$RW_DIR/tmp"
 
 if [ "$DEMO" = "true" ]; then
     python3 /scripts/configure_spilo.py patroni pgqd certificate pam-oauth2
-elif python3 /scripts/configure_spilo.py all; then
-    su postgres -c "PATH=$PATH /scripts/patroni_wait.sh -t 3600 -- envdir $WALE_ENV_DIR /scripts/postgres_backup.sh $PGDATA $BACKUP_NUM_TO_RETAIN" &
+else
+    if [ "$(id -u)" -ne 0 ]; then
+        python3 /scripts/configure_spilo.py all
+        PATH=$PATH /scripts/patroni_wait.sh -t 3600 -- envdir $WALE_ENV_DIR /scripts/postgres_backup.sh $PGDATA $BACKUP_NUM_TO_RETAIN &
+    else
+        python3 /scripts/configure_spilo.py all
+        su postgres -c "PATH=$PATH /scripts/patroni_wait.sh -t 3600 -- envdir $WALE_ENV_DIR /scripts/postgres_backup.sh $PGDATA $BACKUP_NUM_TO_RETAIN" &
+    fi
```

Comment:
Here you are changing the logic.
The original code is checking exit code of configure_spilo.py

Reply:
The ` $BACKUP_NUM_TO_RETAIN` argument could be dropped (I totally forgot about it).

Reply:
Will fix the 'elif' logic and drop `$BACKUP_NUM_TO_RETAIN`